src/compiler/crystal/codegen/target.cr

```diff
@@ -192,6 +192,10 @@ class Crystal::Codegen::Target
     @architecture == "avr"
   end
 
+  def eabi?
+    environment_parts.includes? "eabi"
```

Comment:
Is this supposed to include `arm-linux-gnueabihf` as well, or only target triples that have exactly `-eabi`?

Reply:
Ah, that might be a good idea, I forgot those triples didn't have "eabi" as it's own word.